### DISCUSSION TITLE
Strong sonuçları

### DISCUSSION THREAD

**Discussion by GnMert** [19/classroom#66] [C]
Hocam sizin örneğinizdeki 2. STRONG verilen özelliklere uymuyor. Kodumun bu yüzden hatalı çıktığını düşünüyorum
![Ekran Alıntısı](https://user-images.githubusercontent.com/130256731/231600771-eda26e68-ec1b-4849-8404-8d222f738ec3.PNG)

**Comment by roktas** [comment]
Nesi uymuyor, onu da yazarsan böyle tur atmayız yazışmada.  Düşüncenizi tüm ayrıntılarıyla ifade edin.

Not. Testleri geçen arkadaşların var.  Nihai amaç testlerden başarıyla geçmek ve bunu yaparken temiz bir kod geliştirmiş olmak.

**Comment by roktas** [comment]
Konudan bağımsız bir not...  Github kullanıcı adını biraz daha özenli seçsen?  `mmggmm1`  (Mevcut ismini hesap ayarlarından rename edebiliyorsun)

**Comment by roktas** [comment]
Kodunu inceledim ve depondaki "Feedback" PR'na not düştüm.  Şöyle bir sorununuz var.  Kendi tarafınızdaki sorunları yeterince analiz etmeden (senin durumunda "Actions" sekmesinden erişebileceğin test raporları) sorunu hemen başka bir yerde arıyorsunuz.  Bak problemin ne kadar açık anlatılmış test çıktısında (sadece bir örnek):

![image](https://user-images.githubusercontent.com/47842/231616377-4fa1a65c-28ad-4b35-973b-98509461af24.png)
